### ISSUE TITLE
Add java -version output to the nightly build download page

### ISSUE THREAD

**Issue by M-Davies** [AdoptOpenJDK/openjdk-website#739]
See https://github.com/AdoptOpenJDK/openjdk-build/issues/367

We are now adding the java version output to the metadata from the builds. Assuming this will now automatically appear in the API (CC @johnoliver ?), please provide the java -version output of each build on the download so users can track it back to a git branch and commit.

**Comment by gdams** [comment]
I think the API might need some changes to expose this, lets see what @johnoliver thinks

**Comment by johnoliver** [comment]
I am not sure what is in the java -version output that is not in the version data
looking at

```
{
    "WARNING": "THIS METADATA FILE IS STILL IN ALPHA DO NOT USE ME",
    "os": "linux",
    "arch": "x64",
    "variant": "hotspot",
    "version": {
        "minor": 0,
        "full_version_output": "openjdk version \"11.0.8\" 2020-07-14\nOpenJDK Runtime Environment AdoptOpenJDK (build 11.0.8+5-202006222334)\nOpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.8+5-202006222334, mixed mode)\n",
        "security": 8,
        "pre": null,
        "adopt_build_number": 0,
        "major": 11,
        "version": "11.0.8+5-202006222334",
        "semver": "11.0.8+5.0.202006222334",
        "build": 5,
        "opt": "202006222334",
  },
    "scmRef": "",
    "version_data": "jdk11u",
    "binary_type": "jdk",
    "sha256": "dbc32955fc283989f57ef832f3398af0cc7de7b9287256077206bb6e023dc693"
}
```

It seems like all the data in the full_version_output is also in version

**Comment by M-Davies** [comment]
It was requested at https://github.com/AdoptOpenJDK/openjdk-build/issues/367 that the version output is displayed at the nightly download page

**Comment by johnoliver** [comment]
I think the problem is that issue was requesting this to tie back a build to a git commit, I am not sure the output of java -version here allows you to do that. I would have thought a way to do this would be to include the git commit hash into the metadata

**Comment by M-Davies** [comment]
@johnoliver See existing issue for that https://github.com/AdoptOpenJDK/openjdk-build/issues/474

**Comment by johnoliver** [comment]
@M-Davies yeah I think that would be the issue to solve as I think that would solve https://github.com/AdoptOpenJDK/openjdk-build/issues/367 but I dont believe adding java -version to the metadata does

**Comment by M-Davies** [comment]
See https://github.com/AdoptOpenJDK/openjdk-build/pull/1959

**Comment by M-Davies** [comment]
Transferring to api repo (see https://github.com/AdoptOpenJDK/openjdk-api-v3/issues/235#issuecomment-655392766)
